src/common/transformations/src/transformations/common_optimizations/mark_precision_sensitive_subgraphs.cpp

```diff
@@ -33,6 +33,8 @@ bool ov::pass::MarkPrecisionSensitiveSubgraphs::run_on_model(const std::shared_p
     }
 
     auto markup_func = [](Node* node) {
+        // when dynamism on GPU will be turned on
+        // need to keep the whole shape subgraph in FP32 not only constants
```

Comment:
Don't think that is a valid statement here. If constant is fp16 the whole subgraph will be fp16

Reply:
these lines are not present anymore